### PR TITLE
Edit Sake Address

### PR DESCRIPTION
**Pull request by therealsakeswap**

### CODE REVIEW

File: spaces/sake/index.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   "decimals": 18,
   "symbol": "SAKE",
   "defaultView": "core",
-  "address": "0x066798d9ef0833ccc719076Dab77199eCbd178b0",
-  "token": "0x066798d9ef0833ccc719076Dab77199eCbd178b0",
+  "address": "0x0A291AF57DEb09efd3760B8Ca7167207bcE019d6",
+  "token": "0x0A291AF57DEb09efd3760B8Ca7167207bcE019d6",
   "core": [
     "0x142151D3b15b8961F2937Cf7880Bcd10c050800C"
   ],
```
Comment (e-asia) [inline spaces/sake/index.json:12]: We have updated the format in `index.json`. 
Please add 2 new parameters `chainId` and `decimals`.
You can refer to the documentation that has been completed. https://docs.snapshot.page/guides/create-a-space#3-change-your-space-metadata

**Review by e-asia — CHANGES_REQUESTED** [review]

File: spaces/sake/index.json (hunk 1 above)
Comment (coolc4) [reply spaces/sake/index.json:12]: @e-asia  the new code have include the 2 new  new parameters, chainId and decimals.
Please help check and merge it,thanks

Comment (therealsakeswap) [reply spaces/sake/index.json:12]: @e-asia  the code have included the 2 new parameters chainId and decimals.
Please help to check and merge it. Thanks.

**Review by e-asia — APPROVED** [review]

File: spaces/sake/index.json (hunk 1 above)
Comment (e-asia) [reply spaces/sake/index.json:12]: @therealsakeswap If you change the base token address your old proposals will not be visible anymore under your space, are you sure you want to go forward with this?

**Comment by therealsakeswap** [comment]
dear bonustrack:
Yes, I am sure I have known the proposals in old token address can't visible anymore under my space. we accept it,please help us to merge the new token address.
Thank you.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Pull request merged**